MAUI/PDF-Viewer/Coordinate-Conversion.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Working with PDF coordinate systems in MAUI PDF Viewer control | Syncfusion
+description: Learn here all about coordinate conversion in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control for precise positioning and enhanced interaction. 
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Working with PDF coordinate systems in .NET MAUI PDF Viewer (SfPdfViewer)
+
+PDF Viewer allows users to obtain the PDF page coordinates relative to the PDF Viewer’s client coordinates and vice versa. It also allows you to obtain the scroll point relative to the PDF page coordinates and bring the given region into view.
+
+## Client rectangle
+
+The MAUI PDF Viewer the dimensions of the client area(viewport) are known as client rectangle, which is the bounds of the control. The client rectangle in the MAUI PDF Viewer is represented by the red rectangle in the following figure.
+
+![Coordinate conversion in .NET MAUI PDF Viewer](Images\Coordinate-Conversion\Client-Rectangle.jpg)
+
+The client rectangle can be obtained by the `ClientRectangle` property of the PDF Viewer. The following code sample explains the same.
+
+{% tabs %}
+{% highlight c# %}
+//Retrieve the coordinates of the PDF Viewer's client area (viewport)
+Rectangle clientRectangle = pdfViewerControl.ClientRectangle;
+{% endhighlight %}
+{% endtabs %}
+
+## Client coordinates
+
+The MAUI PDF Viewer specifies the position of points in the client area(viewport) using client coordinates. The upper-left corner of the client area (viewport) of the control is the origin for client coordinates. The client coordinates in the MAUI PDF Viewer is shown in the following figure.
+
+![Coordinate conversion in .NET MAUI PDF Viewer](Images\Coordinate-Conversion\Client-Coordinates.jpg)
+
+### Get page number from client point
+
+You can obtain the page number of the client point using the `GetPageNumberFromClientPoint(Point clientPoint)` method by passing the client point as input parameters. The following code sample explains how to get the page number from the client point.
+
+{% tabs %}
+{% highlight c# %}
+//Subscribe the Tapped event of PDF Viewer
+pdfViewerControl.Tapped += PdfViewerControl_Tapped; 
+ 
+// Handle the tapped event.
+private void PdfViewerControl_Tapped(object sender, GestureEventArgs e)
+{
+    //Retrieve the tapped client area position.
+    Point clientPoint = e.Position;
+    //Retrieve the page number that corresponds to the client point.
+    int pageNumber = pdfViewerControl.GetPageNumberFromClientPoint(clientPoint);
```

Comment:
**pdfViewerControl**

use the uniform object name of PDF Viewer which we used in the getting started page of MAUI PDF Viewer.

Reply:
addressed